### PR TITLE
ci: PR deployment to netlify

### PR DESCRIPTION
**Pull request by tpluscode**
This will publish the built specs as `(branchName--)hydra-extensions.netlify.app`

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -22,3 +22,22 @@ jobs:
         jekyll: false
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        
+    - uses: nelonoel/branch-name@v1.0.1
+    - name: Publish preview
+      uses: jsmrcaga/action-netlify-deploy@v1.5.0
+      with:
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+          NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
```
Comment (tpluscode) [inline .github/workflows/build.yaml:27]: @alien-mcl please add these two variables as repository secrets

`NETLIFY_SITE_ID` can be found on https://app.netlify.com/sites/hydra-extensions/settings/general (**API ID**)
`NETLIFY_AUTH_TOKEN` can be added on https://app.netlify.com/user/applications

Comment (alien-mcl) [reply .github/workflows/build.yaml:27]: Done

Comment (tpluscode) [reply .github/workflows/build.yaml:27]: 👌 

https://netlify-previews--hydra-extensions.netlify.app/shacl/

**Review by alien-mcl — APPROVED** [review]
We need some landing page to the resulting website.

**Pull request merged**